rubin_sim/scheduler/basis_functions/mask_basis_funcs.py

```diff
@@ -19,6 +20,29 @@
 ]
 
 
+class SolarElongMaskBasisFunction(BaseBasisFunction):
+    """Mask regions beyond some solar elongation
```

Comment:
Could you please make it clear in the docstring if "beyond some solar elongation" means "closer to the sun" (smaller values) or "farther from the sun" (larger values)? 
I would have thought that it might be reasonable to add both upper and lower limits to the solar elongation mask? 
(don't have to set them both)